### PR TITLE
Hook refactor

### PR DESCRIPTION
**Pull request by carflor**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features

### Detailed Description
- fixed back button convention to a home button with title 
- addressed lighthouse score and now it is 100%
- hover fixed for painter name and for home btn
- loading message styled
- build test for custom hook for fetching painting summary 
### Files modified:
- [ ] App.js
- [ ] App.test.js
- [ ] App.css
- [ ] Gallery.js
- [ ] Gallery.test.js
- [x] Gallery.css
- [ ] Painting.js
- [ ] Painting.test.js
- [ ] Painting.css
- [ ] PaintingInfo.js
- [ ] PaintingInfo.test.js
- [x] PaintingInfo.css
- [x] PainterInfo.css

### CODE REVIEW

File: src/Hooks/usePaintingSummary.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import {useState, useEffect} from 'react';
+
+const usePaintingSummary = (contentId) => {
```
Comment (cwefso) [inline src/Hooks/usePaintingSummary.js:3]: Beautiful custom hook

File: src/Hooks/usePaintingSummary.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+  
+import { renderHook } from '@testing-library/react-hooks';
+import usePaintingSummary from './usePaintingSummary';
+// import TestRenderer from 'react-test-renderer';
+
+
+describe('the usePaintingSummary hook', () => {
```
Comment (cwefso) [inline src/Hooks/usePaintingSummary.test.js:7]: This looks really good. I don't know what else we're supposed to be testing here.

Comment (Josephhaefling) [reply src/Hooks/usePaintingSummary.test.js:7]: Nice job with the testing!

File: src/Hooks/usePaintingSummary.js (hunk 1 above)
Comment (Josephhaefling) [reply src/Hooks/usePaintingSummary.js:3]: The page looks great. I like getting rid of the back button. It looks very clean this way.

**Pull request merged**